### ISSUE TITLE
plotAbundance error?

### ISSUE THREAD

**Issue by antagomir** [microbiome/miaViz#44]
The `plotAbundance` method provides the `features` argument. 

This is intended to highlight colData variables below the plot. According to manpage:

> "features: a single ‘character’ value defining a column from ‘colData’
>           to be plotted below the abundance plot. Continuous numeric
>           values will be plotted as point, whereas factors and
>           character will be plotted as colour-code bar. (default:
>           ‘features = NULL’)"
> 

The following commands output ggplot object p, which is not a single figure with colData highlighted below the figure. Instead, this outputs two different figures. My impression is that either the function or the manpage has some mistake. The expected behaviour (to me) would be something like that explained on the manpage.

```
data(GlobalPatterns)
se <- GlobalPatterns
p <- plotAbundance(se, abund_values = "counts", rank = "Phylum", features = "SampleType")
names(p)
```

[1] "abundance"  "SampleType"

On the same go, I would like to see an example in manpages showing how one can add sampleIDs below the plot. This would be a common use case for this.

**Comment by antagomir** [comment]
I suggest that @YagmurSimsekk could check this, with support from @TuomasBorman

**Comment by FelixErnst** [comment]
TL;DR: This works as intended. I won't accept the PR, since it forces the combination of plots by default.

The man page very very clearly states:
```
Value
a ggplot object or list of two ggplot objects, if features are added to the plot.
```
So the function and the man page are both correct. 

I added an additional example for producing the result, I think you are looking for:
```r
library(miaViz)
library(patchwork)
data(GlobalPatterns)
se <- GlobalPatterns
plot <- plotAbundance(se, abund_values = "counts", rank = "Phylum", features = "SampleType")
plot[[1]] <- plotAbundance(se, abund_values = "counts", rank = "Phylum",
                           features = "SampleType", add_legend = FALSE,
                           add_x_text = TRUE)[[1]] +
                           theme(axis.text.x = element_text(angle = 90)) 
wrap_plots(plot, ncol = 1, heights = c(0.8,0.2))
``` 
Does this fit, what you are looking for?

**Comment by antagomir** [comment]
Yes, this solution works as well, good examples are always valuable.
